src/server/index.spec.tsx

```diff
@@ -697,6 +697,32 @@ describe(__filename, () => {
         expect(response.status).toEqual(200);
         expect(response.header).toHaveProperty('content-security-policy');
 
+        const policy = cspParser(response.header['content-security-policy']);
+        expect(policy['script-src']).toEqual([
+          "'self'",
+          "'unsafe-inline'",
+          `${devEnv.REACT_APP_ANALYTICS_HOST}${ANALYTICS_PATH}`,
+        ]);
+      });
+
+      it('uses debug analytics path when REACT_APP_GA_DEBUG_MODE is true', async () => {
+        const content = '<h1>It works!</h1>';
+        fakeCreateReactAppServerApp.get('/*', (req, res) => res.send(content));
+
+        const server = request(
+          createServer({
+            env: {
+              ...devEnv,
+              REACT_APP_GA_DEBUG_MODE: 'true',
+            } as ServerEnvVars,
+            rootPath: fixturesPath,
+          }),
+        );
+
+        const response = await server.get('/');
+        expect(response.status).toEqual(200);
+        expect(response.header).toHaveProperty('content-security-policy');
```

Comment:
I'm not sure what's being tested here exactly - it seems the same to the other surrounding tests.

Reply:
It's really just the assertion below about a specific `content-security-policy` that we care about, but these guard assertions will make a test failure easier to debug if one happens, so I can see value in them. Do you feel strongly that they should be removed?

Reply:
guard assumptions are fine, but I couldn't identify what was uniquely being tested in this test versus the other tests that seemed to share the same code.